hexl/experimental/seal/key-switch-internal.cpp

```diff
@@ -11,6 +11,7 @@
 #include "hexl/eltwise/eltwise-fma-mod.hpp"
 #include "hexl/eltwise/eltwise-mult-mod.hpp"
 #include "hexl/eltwise/eltwise-reduce-mod.hpp"
+#include "hexl/experimental/seal/get-ntt.hpp"
```

Comment:
Can we change the name of this library to something like ntt-objs.hpp or ntt-cache.hpp?